src/Settings/TypeSettings.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import { stripesShape } from '@folio/stripes/core';
+import { Select } from '@folio/stripes/components';
+import { ControlledVocab } from '@folio/stripes/smart-components';
+import TypeStatus from '../Utils/TypeStatus';
```

Comment:
why is it called like a class? additionally we use named exports in our modules

Reply:
Thanks! Changed it to a named export.